packages/task/src/node/process/process-task-runner.ts

```diff
@@ -89,23 +88,28 @@ export class ProcessTaskRunner implements TaskRunner {
         };
 
         try {
-            // use terminal or raw process
-            let proc: TerminalProcess | RawProcess;
             const processType = taskConfig.type === 'process' ? 'process' : 'shell';
+            let proc: Process;
+
+            // Always spawn a task in a pty, the only difference between shell/process tasks is the
+            // way the command is passed:
+            // - process: directly look for an executable and pass a specific set of arguments/options.
+            // - shell: defer the spawning to a shell that will evaluate a command line with our executable.
             if (processType === 'process') {
-                this.logger.debug('Task: creating underlying raw process');
-                proc = this.rawProcessFactory(<RawProcessOptions>{
-                    command: command,
-                    args: args,
-                    options: options
+                proc = this.terminalProcessFactory(<TerminalProcessOptions>{
```

Comment:
Was it voluntary to remove the debug log here but keep the equivalent in the "else" block?

Reply:
I think I can get rid of both?

Not sure this is useful information?

Reply:
Or I can add the other one back, will do that.

Reply:
ok to remove both as long as we're consistent